spec/rails_helper.rb

```diff
@@ -76,4 +76,13 @@
   config.after(:each, stripe_mock: true) do
     StripeMock.stop
   end
+
+  # remove with_advisory_lock lock files created during running specs
+  config.before do
+    ENV["FLOCK_DIR"] = Dir.mktmpdir
+  end
+
+  config.after do
+    FileUtils.remove_entry_secure ENV["FLOCK_DIR"]
+  end
```

Comment:
The line of code following the comment doesn't actually do what the line describes?
```suggestion
  config.before do
    ENV["FLOCK_DIR"] = Dir.mktmpdir
  end

  config.after do
    # Remove 'with_advisory_lock' lock files created during running specs.
    FileUtils.remove_entry_secure ENV["FLOCK_DIR"]
  end
```